tests/impact/client/fixtures.py

```diff
@@ -22,6 +23,42 @@
 MockedServer = collections.namedtuple('MockedServer', ['url', 'context', 'adapter'])
 ExperimentMock = collections.namedtuple('ExperimentMock', ['entity', 'service'])
 WorkspaceMock = collections.namedtuple('WorkspaceMock', ['entity', 'service'])
+ProjectMock = collections.namedtuple('ProjectMock', ['entity', 'service'])
+TEST_WORKSPACE_DEFINITION = {

```

Comment:
given how this variable is used it would probably be better to have it like a function:
```
def get_test_workspace_definition(name="newWorkspace"):
  return {
    ...
  }
```
As to not have to creating, copy, and then mutate it.